CONTRIBUTING.md

```diff
@@ -1,69 +1,216 @@
-
-## Supported formats
-
-Our documentation includes both Markdown and RST files. Markdown is encouraged
-over RST due to the lower barrier to participation. GitHub-flavored Markdown is preferred
-for all submissions as it renders accurately on our GitHub repositories. For existing documentation,
-[MyST](https://myst-parser.readthedocs.io/en/latest/intro.html) Markdown
-is used to implement certain features unsupported in GitHub Markdown. This is
-not encouraged for new documentation. AMD will transition
-to stricter use of GitHub-flavored Markdown with a few caveats. ROCm documentation
-also uses [Sphinx Design](https://sphinx-design.readthedocs.io/en/latest/index.html)
-in our Markdown and RST files. We also use Breathe syntax for Doxygen documentation
-in our Markdown files. See
-[GitHub](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github)'s
-guide on writing and formatting on GitHub as a starting point.
-
-ROCm documentation adds additional requirements to Markdown and RST based files
-as follows:
-
-* Level one headers are only used for page titles. There must be only one level
-  1 header per file for both Markdown and Restructured Text.
-* Pass [markdownlint](https://github.com/markdownlint/markdownlint) check via
-  our automated GitHub action on a pull request (PR).
-  See the {doc}`rocm-docs-core linting user guide <rocm-docs-core:user_guide/linting>` for more details.
-
-## Filenames and folder structure
-
-Please use kebab-case (all lower case letters and dashes instead of spaces)
-for file names. For example, `example-file-name.md`.
-Our documentation follows Pitchfork for folder structure.
-All documentation is in `/docs` except for special files like
-the contributing guide in the `/` folder. All images used in the documentation are
-placed in the `/docs/data` folder.
+AMD values and encourages contributions to our code and documentation. If you choose to
+contribute, we encourage you to be polite and respectful. Improving documentation is a long-term
+process, to which we are dedicated.
 
-## Language and style
+If you have issues when trying to contribute, refer to the
+[discussions](https://github.com/RadeonOpenCompute/ROCm/discussions) page in our GitHub
+repository.
+
+## Folder structure and naming convention
+
+Our documentation follows the Pitchfork folder structure. Most documentation files are stored in the
+`/docs` folder. Some special files (such as release, contributing, and changelog) are stored in the root
+(`/`) folder.
+
+All images are stored in the `/docs/data` folder. An image's file path mirrors that of the documentation
+file where it is used.
+
+Our naming structure uses kebab case; for example, `my-file-name.rst`.
+
+## Supported formats and syntax
+
+Our documentation includes both Markdown and ReST files. We are gradually transitioning existing
+Markdown to ReST in order to more effectively meet our documentation needs. When contributing,
```

Comment:
ReST? or RST? @LisaDelaney

Reply:
Hm, the internet says RST, ReST, and reST are all valid. Sphinx uses reST, so maybe we should follow suit? Seems logical to me, given that it's called reStructuredText. I like reST. Thoughts?

Reply:
reST is contextually correct; however, it may cause confusion with REST APIs. Which is why we used RST. Am ok with reST, but prefer RST.

Reply:
Sure, that makes sense. I'm okay with RST.